proto/streamlit/proto/Block.proto

```diff
@@ -56,6 +56,7 @@ message Block {
       BOTTOM = 2;
     }
     VerticalAlignment vertical_alignment = 3;
+    bool border = 4;
```

Comment:
Similar to the comment [here](https://github.com/streamlit/streamlit/pull/9928#discussion_r1867673984): I would indicate in the variable name that this is of boolean type